examples/amm/src/state.rs

```diff
@@ -1,8 +1,35 @@
-use linera_sdk::views::{RegisterView, ViewStorageContext};
+use amm::SwapBidId;
+use async_graphql::SimpleObject;
+use fungible::AccountOwner;
+use linera_sdk::{
+    base::Amount,
+    views::{MapView, RegisterView, ViewStorageContext},
+};
 use linera_views::views::{GraphQLView, RootView};
+use serde::{Deserialize, Serialize};
 
 #[derive(RootView, GraphQLView)]
 #[view(context = "ViewStorageContext")]
 pub struct Amm {
-    pub will_remove: RegisterView<u64>,
+    // The next_swap_bid_number contains the swap_bid_id so that
+    // it gets created from 0, to infinity.
```

Comment:
Let's make these rustdoc comments. Even on private items it doesn't hurt to have comments in a shape that could be turned into documentation at any time. And this one is public anyway.

```suggestion
    /// The `next_swap_bid_number` contains the `swap_bid_id` so that
    /// it gets created from 0, to infinity.
```